client/src/components/App.test.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {mount, shallow} from 'enzyme';
+
+import App from './App';
+
+xdescribe('App', () => {
```

Comment:
Do you have the x in front because you do not want these to run just yet? Have any tests been able to pass? Also, what is the file structure like with tests for you? Do you have a tests folder or are you just creating test files next to the file they are testing?

Reply:
I'm currently creating tests in the same folder as my components, and naming the test files after the component file they're testing, with `.test.js` appended.